catroid/src/org/catrobat/catroid/physics/content/bricks/CollisionReceiverBrick.java

```diff
@@ -63,6 +63,10 @@ public CollisionReceiverBrick(String spriteName) {
 	public CollisionReceiverBrick(CollisionScript collisionScript) {
 		this.collisionScript = collisionScript;
 		this.selectedMessage = "";
+
+		if (collisionScript != null && collisionScript.isCommentedOut()) {
+			setCommentedOut(true);
+		}
 	}
```

Comment:
Why is the clone method not altered in order to set the cloned brick comment state ?